gio-subclass/src/auto/action.rs

```diff
@@ -0,0 +1,238 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ cf72ed8+)
+// from gir-files (https://github.com/gtk-rs/gir-files @ 47c69e6)
+// DO NOT EDIT
+
+#[allow(unused_imports)]
+use libc::{c_int, c_char, c_uchar, c_float, c_uint, c_double,
+    c_short, c_ushort, c_long, c_ulong,
+    c_void, size_t, ssize_t, intptr_t, uintptr_t, time_t, FILE};
+
+#[allow(unused_imports)]
+use glib_ffi::{gboolean, gconstpointer, gpointer, GType};
+
+#[cfg(any(feature = "v2_38", feature = "dox"))]
+use Error;
+use gio;
+use gio_ffi;
+use glib;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+use gobject_subclass::anyimpl::*;
+use gobject_subclass::object::*;
+
+
+pub trait ActionImpl: AnyImpl + 'static {
```

Comment:
Also needs to be `ObjectImpl`. The docs say that `GAction requires GObject` (this should also be machine-readable from the gir file)

Reply:
True. For interfaces I did not expect there to be requirements as auch

Reply:
This does pose a problem though. ```ObjectImpl``` is generic over T, but ```ActionImpl``` isn't?

Reply:
It'll probably break a lot, but I don't see any other option? Though I'm not sure what traits ```T``` should be bound to.

Reply:
Would have to be bound to `ObjectImpl`. Let's worry about that in a separate PR